src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/CountryLists.java

```diff
@@ -0,0 +1,288 @@
+            oeTempList.add("Saint Kitts and Nevis");
+            oeTempList.add("Saint Lucia");
+            oeTempList.add("Saint Martin (French part)");
+            oeTempList.add("Saint Pierre and Miquelon");
+            oeTempList.add("Saint Vincent and the Grenadines");
+            oeTempList.add("Samoa");
+            oeTempList.add("San Marino");
+            oeTempList.add("Sao Tome and Principe");
+            oeTempList.add("Sark");
+            oeTempList.add("Saudi Arabia");
+            oeTempList.add("Senegal");
+            oeTempList.add("Serbia");
+            oeTempList.add("Seychelles");
+            oeTempList.add("Sierra Leone");
+            oeTempList.add("Singapore");
+            oeTempList.add("Sint Maarten (Dutch part)");
+            oeTempList.add("Slovakia");
+            oeTempList.add("Slovenia");
+            oeTempList.add("Solomon Islands");
+            oeTempList.add("Somalia");
+            oeTempList.add("South Africa");
+            oeTempList.add("South Georgia and the South Sandwich Islands");
+            oeTempList.add("South Korea");
+            oeTempList.add("South Sudan");
+            oeTempList.add("Spain");
+            oeTempList.add("Sri Lanka");
+            oeTempList.add("Sudan");
+            oeTempList.add("Suriname");
+            oeTempList.add("Svalbard and Jan Mayen");
+            oeTempList.add("Sweden");
+            oeTempList.add("Switzerland");
+            oeTempList.add("Syria");
+            oeTempList.add("Taiwan");
+            oeTempList.add("Tajikistan");
+            oeTempList.add("Tanzania");
+            oeTempList.add("Thailand");
+            oeTempList.add("Togo");
+            oeTempList.add("Tokelau");
+            oeTempList.add("Tonga");
+            oeTempList.add("Trinidad and Tobago");
+            oeTempList.add("Tunisia");
+            oeTempList.add("Turkey");
+            oeTempList.add("Turkmenistan");
+            oeTempList.add("Turks and Caicos Islands");
+            oeTempList.add("Tuvalu");
+            oeTempList.add("Uganda");
+            oeTempList.add("Ukraine");
+            oeTempList.add("United Arab Emirates");
+            oeTempList.add("United States");
+            oeTempList.add("United States Minor Outlying Islands");
+            oeTempList.add("Uruguay");
+            oeTempList.add("Uzbekistan");
+            oeTempList.add("Vanuatu");
+            oeTempList.add("Vatican City");
+            oeTempList.add("Venezuela");
+            oeTempList.add("Vietnam");
+            oeTempList.add("Virgin Islands, British");
+            oeTempList.add("Virgin Islands, U.S.");
+            oeTempList.add("Wallis and Futuna");
+            oeTempList.add("Western Sahara");
+            oeTempList.add("Yemen");
+            oeTempList.add("Zambia");
+            oeTempList.add("Zimbabwe");
+            overseasEntityCountryList = Collections.unmodifiableList(oeTempList);
+        }
+
+        static {
```

Comment:
No need for three static blocks, just combine into one.

Reply:
Using a single block